onnxruntime/core/providers/cuda/nn/batch_norm.h

```diff
@@ -36,6 +41,7 @@ class BatchNorm final : public CudaKernel {
   double epsilon_;
   int64_t spatial_ = 1;  // default as per spec
   cudnnBatchNormMode_t cudnn_batch_norm_mode_;
+  double momentum_;
 };
```

Comment:
Should this be explicitly initialized as it's only conditionally set in the ctor?